src/PrestaShopBundle/Resources/config/routing/admin/improve/design/theme.yml

```diff
@@ -12,27 +12,33 @@ admin_themes_upload_logos:
   defaults:
     _controller: PrestaShopBundle:Admin\Improve\Design\Theme:uploadLogos
     _legacy_controller: AdminThemes
+    _legacy_link: AdminThemes:submitOptionsconfiguration
 
 admin_themes_export_current:
   path: /export
   methods: [GET]
   defaults:
     _controller: PrestaShopBundle:Admin\Improve\Design\Theme:export
     _legacy_controller: AdminThemes
+    _legacy_link: AdminThemes:exporttheme
 
 admin_themes_import:
   path: /import
   methods: [GET, POST]
   defaults:
     _controller: PrestaShopBundle:Admin\Improve\Design\Theme:import
     _legacy_controller: AdminThemes
+    _legacy_link: AdminThemes:importtheme
 
 admin_themes_enable:
   path: /{themeName}/enable/
```

Comment:
```suggestion
  path: /{themeName}/enable
```

Reply:
I'm not sure this changes something 🤔AFAIK Symfony does not care about trailing slashes, and the route below `path: /{themeName}/delete/` has a trailing slash but LegacyUrlConverter test works just fine

Reply:
it does, `/` at the end means directory.
If you try to go to http://ps-develop.localhost/admin-dev/index.php/sell/catalog/products/
it'll not work :D

Reply:
So we should modify `/{themeName}/delete/` into `/{themeName}/delete` as well ?

Reply:
Yep!
Never end with a slash if it's not a directory :)